tests/test_pr_merge_readiness_gate.py

```
@@ -110,6 +114,195 @@ def test_standalone_actionable_review_summary_still_requires_mapping() -> None:
     assert merge_gate._covered_review_summary_urls([summary], set()) == set()
 
 
+def test_canonical_artifact_link_count_requires_true_markdown_destination() -> None:
+    body = """
+Plain text: docs/review/PR_42_FIXED_MAPPING.md
+[canonical mapping](docs/review/PR_42_FIXED_MAPPING.md)
+\\[escaped literal](docs/review/PR_42_FIXED_MAPPING.md)
+    [indented code](docs/review/PR_42_FIXED_MAPPING.md)
+	[tab-indented code](docs/review/PR_42_FIXED_MAPPING.md)
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)